.github/workflows/pr-build.yml

```diff
@@ -34,14 +34,13 @@ jobs:
       - name: Install toolchain
         uses: actions-rs/toolchain@v1
         with:
-          profile: minimal
           toolchain: nightly
           target: wasm32-unknown-unknown
           default: true
       - name: Check
-        run: cargo check --all
+        run: cargo update -p environmental && cargo check --features with-bifrost-runtime
```

Comment:
remove command `cargo update -p environmental` due to that package is updated in Cargo.lock